doc/source/parameters/unresolved_cfd-dem/cfd_dem.rst

```diff
@@ -0,0 +1,28 @@
+***********************************************
+CFD-DEM
+***********************************************
+This subsection includes parameters related to multiphase flow simulations using the both the gls_vans solver and the cfd-dem solver within Lethe.
```

Comment:
the cfd-dem solver's name is cfd_dem_coupling, so I think it's better to use the exact name instead of cfd-dem here.